File: javascript/control_flow/if_statement/if_else_statement.js
#!/usr/bin/env node

if (1 > 2)
  console.log('1 > 2');
else
  console.log('!(1 > 2)');

if (2 > 1)
  console.log('2 > 1');
else
  console.log('!(2 > 1)');

console.log()

condition = false;

if (condition) {
  console.log('false: ok');
} else {
  console.log('false: nope');
}

condition = true;

if (condition) {
  console.log('true: ok');
} else {
  console.log('true: nope');
}

console.log();

File: ruby/control_flow/if_statement/if_else_statement.rb
#!/usr/bin/ruby

if 1 > 2
    puts '1 > 2'
else
    puts '!(1 > 2)'
end

if 2 > 1
    puts '2 > 1'
else
    puts '!(2 > 1)'
end

puts

condition = false

if condition
  puts 'false: ok'
else
  puts 'false: nope'
end

condition = true

if condition
  puts 'true: ok'
else
  puts 'true: nope'
end

puts
